acos_client/v21/vrrp_a/vrrp_global.py

```diff
@@ -16,16 +16,14 @@
 
 from acos_client.v21 import base
 
-import six
-
 
 class VRRPAGlobal(base.BaseV21):
     def get(self, **kwargs):
         return self._get("vrrp_a.get", **kwargs)
 
-    def set(self, status, device_id, set_id, default_vrid, hello_interval, dead_timer, 
-            track_event_delay, preemption_delay, arp_retry, 
-            vrid_list={}, 
+    def set(self, status, device_id, set_id, default_vrid, hello_interval, dead_timer,
+            track_event_delay, preemption_delay, arp_retry,
+            vrid_list={},
```

Comment:
`vrid_list`=`None` here and below
```
if vrid_list is None:
    vrid_list = []
```
This is to avoid undesired behavior with persistence in `vrid_list` beyond a single method call.

Reply:
Agreed, but this was not my code and out of scope for this PR.  There are many, many things that need to be fixed in this repo.  :-)